python/tvm/relay/op/strategy/generic.py

```diff
@@ -715,8 +715,14 @@ def dilation2d_strategy(attrs, inputs, out_type, target):
     return strategy
 
 
+def maybe_copy_tensor_b(tensor_a, tensor_b):
```

Comment:
I'm wondering if we could have a better name (with docstring) for this utility. Something like `copy_if_identical`?

Reply:
Yeah I didn't put much thought into the name, I like your suggestion. I'll also add a comment explaining why this is necessary.